configure

```diff
@@ -4643,7 +4658,7 @@ else
     We can't simply define LARGE_OFF_T to be 9223372036854775807,
     since some C++ compilers masquerading as C compilers
     incorrectly reject 9223372036854775807.  */
-#define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
+#define LARGE_OFF_T ((((off_t) 1 << 31) << 31) - 1 + (((off_t) 1 << 31) << 31))
```

Comment:
@guyharris is this an error, or something we should be doing?  
This seems to have jumped into this patch, and I don't know why.

Reply:
This might just be a case of different versions of autoconf generating different code.